src/main/java/com/notably/logic/parser/DeleteCommandParser.java

```diff
@@ -33,12 +33,17 @@ public DeleteCommandParser(Model notablyModel) {
         ArgumentMultimap argMultimap =
                 ArgumentTokenizer.tokenize(args, PREFIX_TITLE);
 
+        String title;
         if (!ParserUtil.arePrefixesPresent(argMultimap, PREFIX_TITLE)
                 || !argMultimap.getPreamble().isEmpty()) {
-            throw new ParseException(String.format("Invalid Command"));
+            title = args.trim();
+            if (title.equals("")) {
```

Comment:
You should actually do `title.isEmpty()`, more idiomatic!

Reply:
will do